CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.13)
+cmake_minimum_required(VERSION 3.14)
```

Comment:
Why is minimum CMake version increased?

Reply:
No particular need, but thought it didn't hurt being on a later version given that I didn't find any problems.
I have changed this back to 3.13